### PR TITLE
fix: for `image create` rename`root-volume` into `snapshot-id`

### PR DESCRIPTION
**Pull request by loicbourgois**

### CODE REVIEW

File: internal/namespaces/instance/v1/custom_image_test.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+)
+
+func Test_ImageCreate(t *testing.T) {
+	t.Run("simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		BeforeFunc: func(ctx *core.BeforeFuncCtx) error {
+			ctx.Meta["Server"] = ctx.ExecuteCmd("scw instance server create image=ubuntu_bionic stopped")
```
Comment (kindermoumoute) [inline internal/namespaces/instance/v1/custom_image_test.go]: Use helper to create the server

File: internal/namespaces/instance/v1/custom_image_test.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		},
+		Cmd: "scw instance image create snapshot-id={{ .SnapshotResponse.Snapshot.ID }} arch=x86_64",
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(0),
+		),
+		AfterFunc: func(ctx *core.AfterFuncCtx) error {
+			ctx.ExecuteCmd("scw instance image delete image-id=" + ctx.CmdResult.(*instance.CreateImageResponse).Image.ID)
+			ctx.ExecuteCmd("scw instance snapshot delete snapshot-id={{ .SnapshotResponse.Snapshot.ID }}")
+			ctx.ExecuteCmd("scw instance server delete server-id={{ .Server.ID }} delete-ip=true delete-volumes=true force-shutdown=true")
```
Comment (kindermoumoute) [inline internal/namespaces/instance/v1/custom_image_test.go]: Use helper to delete the server

File: internal/namespaces/instance/v1/instance_cli_test.go [hunk 3]
```diff
@@ -316,3 +316,26 @@ func Test_ServerUpdate(t *testing.T) {
...
 }
+
+//
+// Snapshot
+//
+func Test_SnapshotCreate(t *testing.T) {
+	t.Run("simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		BeforeFunc: func(ctx *core.BeforeFuncCtx) error {
+			ctx.Meta["Server"] = ctx.ExecuteCmd("scw instance server create image=ubuntu_bionic stopped")
```
Comment (kindermoumoute) [inline internal/namespaces/instance/v1/instance_cli_test.go]: Use helper to create the server

File: internal/namespaces/instance/v1/instance_cli_test.go [hunk 4]
```diff
@@ -316,3 +316,26 @@ func Test_ServerUpdate(t *testing.T) {
...
+			return nil
+		},
+		Cmd: `scw instance snapshot create volume-id={{ (index .Server.Volumes "0").ID }}`,
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(0),
+		),
+		AfterFunc: func(ctx *core.AfterFuncCtx) error {
+			ctx.ExecuteCmd("scw instance snapshot delete snapshot-id=" + ctx.CmdResult.(*instance.CreateSnapshotResponse).Snapshot.ID)
+			ctx.ExecuteCmd("scw instance server delete server-id={{ .Server.ID }} delete-ip=true delete-volumes=true force-shutdown=true")
```
Comment (kindermoumoute) [inline internal/namespaces/instance/v1/instance_cli_test.go]: Use helper to delete the server

File: internal/namespaces/instance/v1/custom_image.go [hunk 5]
```diff
@@ -93,3 +94,28 @@ func imageListBuilder(c *core.Command) *core.Command {
...
+
+	c.ArgsType = reflect.TypeOf(CreateImageRequestCustom{})
+
+	c.Run = func(ctx context.Context, args interface{}) (i interface{}, e error) {
+		requestCustom := args.(*CreateImageRequestCustom)
+
+		request := requestCustom.CreateImageRequest
+		request.RootVolume = requestCustom.SnapshotID
+
+		client := core.ExtractClient(ctx)
```
Comment (QuentinBrosse) [inline internal/namespaces/instance/v1/custom_image.go]: Can you use the old run instead of these 3 lines? It's better for maintainability.

**Pull request merged**